src/plugins/saved_objects/README.md

```diff
@@ -85,14 +86,71 @@ class SavedVis extends SavedObjectClass {
         afterOpenSearchResp: async (savedObject: SavedObject) => {
           const savedVis = (savedObject as any) as ISavedVis;
           ... ...
-          
+
           return (savedVis as any) as SavedObject;
         },
 ```
 
+2. Optionally create a loader class that extends `SavedObjectLoader`. This can be useful for performing default CRUD operations on this particular saved object type, as well as overriding default utility functions like `find`. For example, the `visualization` saved object overrides `mapHitSource` (used in `find` & `findAll`) to do additional checking on the returned source object, such as if the returned type is valid:
```

Comment:
nit: 

```suggestion
2. Optionally, create a loader class that extends `SavedObjectLoader`. This can be useful for performing default CRUD operations on this particular saved object type and overriding default utility functions like `find`. For example, the `visualization` saved object overrides `mapHitSource` (used in `find` and `findAll`) to perform additional checks on the returned source object, such as checking if the returned type is valid:
```